setup.cfg

```diff
@@ -1,5 +1,5 @@
 [metadata]
-name = hip
+name = Hip
```

Comment:
I'd consider this field a package name field so would be "hip" (ie pip install hip)

Reply:
Great catch, thanks!